pkg/util/util.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"github.com/blang/semver"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("storageos.util")
```

Comment:
This is fine but I think we usually do something like `.WithName("storageos").WithName("util")` for the same result.

Reply:
I will fix it, just for your info: https://github.com/storageos/cluster-operator/pull/319/files#diff-eacdd7d7d1fc5c3620908f182ce14bb70864ff452d9b04ec9dc6c8fbea44f3a8L99 I followed the existing example.

Reply:
:smile: old code